app.js

```diff
@@ -0,0 +1,77 @@
+
+console.log("Hello, World!"); //->English
+console.log("Silav, Cihan!");//->Kurdish
+console.log("Hoi, Wereld!"); //->Dutch
+
+console.log("I'm awesome!");
+
+let x =
+console.log('I think the value of x is bigger than 5');
+console.log("The value of x is 6");
+let x = 6;
+console.log(I think the value of x is 6);
+console.log(x);
+
+let y ="Sivan";
+console.log("I think the value of y is a name");
+console.log(y);
+const y = "Hassam";
+console.log("There will be an error because now I have
+             the same variable with different values.");
+console.log(y);
+
+const z = 7.25;
+console.log(z);
+const a = z;
+console.log(a);
+var round =Math.round(7.25);
+console.log(round)
+
+let arrays =

```

Comment:
Same here as above, I don't think this will compile

Reply:
Changed to : let arrays;